diff --git a/test/e2e-common.sh b/test/e2e-common.sh
index 220aa5b..4dcdc33 100755
--- a/test/e2e-common.sh
+++ b/test/e2e-common.sh
@@ -9,6 +9,7 @@ export SYSTEM_NAMESPACE=${SYSTEM_NAMESPACE:-"knative-eventing"}
 export ISTIO_NAMESPACE=${ISTIO_NAMESPACE:-"istio-system"}
 export KAFKA_BROKER_TEMPLATES=${KAFKA_BROKER_TEMPLATES:-"${REPO_ROOT_DIR}/test/e2e/templates/kafka-broker"}
 export KAFKA_NAMESPACED_BROKER_TEMPLATES=${KAFKA_BROKER_TEMPLATES:-"${REPO_ROOT_DIR}/test/e2e/templates/kafka-namespaced-broker"}

 source "${REPO_ROOT_DIR}"/vendor/knative.dev/hack/e2e-tests.sh
 
@@ -25,6 +26,7 @@ function run_eventing_core_tests() {
     -parallel=12 \
     -run TestPingSource \
     ./test/rekt/ \
+     --images.producer.file="${REPO_ROOT_DIR}/openshift/images.yaml" \
     --istio.enabled=true || return $?
 
   BROKER_TEMPLATES="${KAFKA_BROKER_TEMPLATES}" go_test_e2e \
@@ -32,6 +34,7 @@ function run_eventing_core_tests() {
     -parallel=12 \
     -run TestBrokerConformance \
     ./test/rekt/ \
+     --images.producer.file="${REPO_ROOT_DIR}/openshift/images.yaml" \
     --istio.enabled=true || return $?
 
   BROKER_TEMPLATES="${KAFKA_NAMESPACED_BROKER_TEMPLATES}" go_test_e2e \
@@ -39,6 +42,7 @@ function run_eventing_core_tests() {
     -parallel=12 \
     -run TestBrokerConformance \
     ./test/rekt/ \
+     --images.producer.file="${REPO_ROOT_DIR}/openshift/images.yaml" \
     --istio.enabled=true || return $?
 
   BROKER_TEMPLATES="${KAFKA_BROKER_TEMPLATES}" go_test_e2e \
@@ -46,6 +50,7 @@ function run_eventing_core_tests() {
     -parallel=12 \
     -run TestContainerSource \
     ./test/rekt/ \
+     --images.producer.file="${REPO_ROOT_DIR}/openshift/images.yaml" \
     --istio.enabled=true || return $?
 
   BROKER_TEMPLATES="${KAFKA_BROKER_TEMPLATES}" go_test_e2e \
@@ -53,6 +58,7 @@ function run_eventing_core_tests() {
     -parallel=12 \
     -run TestSinkBinding \
     ./test/rekt/ \
+     --images.producer.file="${REPO_ROOT_DIR}/openshift/images.yaml" \
     --istio.enabled=true || return $?
 
   popd
@@ -66,6 +72,7 @@ function run_eventing_kafka_broker_tests() {
     -parallel=12 \
     -run TestKafkaSource \
     ./test/e2e_new/... \
+     --images.producer.file="${REPO_ROOT_DIR}/openshift/images.yaml" \
     --istio.enabled=true || return $?
 
   BROKER_TEMPLATES="${KAFKA_BROKER_TEMPLATES}" BROKER_CLASS="Kafka" go_test_e2e \
@@ -73,6 +80,7 @@ function run_eventing_kafka_broker_tests() {
     -parallel=12 \
     -run TestKafkaSink \
     ./test/e2e_new/... \
+     --images.producer.file="${REPO_ROOT_DIR}/openshift/images.yaml" \
     --istio.enabled=true || return $?
 
   popd
